ConsistencyManager/ConsistencyManager.swift

```diff
@@ -140,23 +140,23 @@ open class ConsistencyManager {
      Note that calling this method on a paused listener will not unpause it.
      - parameter listener: The consistency manager listener that is listening to a model
     */
-    open func listenForUpdates(_ listener: ConsistencyManagerListener) {
+    open func addListener(_ listener: ConsistencyManagerListener) {
```

Comment:
`add(listener: ConsistencyManagerListener)` ?

Same could be done with the other methods pause/resume (and other methods in this PR).

Unsure if this is better or more Swifty. Style guide explicitly prefers your method when the type information is weak (but that isn't the case here). Another counter example is `addSubview(_ view: UIView)` but I wonder if that is just because that would be a massive change.

Up to you.